lego/apps/surveys/models.py

```diff
@@ -15,6 +22,31 @@ class Survey(BasisModel):
     )
     event = models.OneToOneField('events.Event', on_delete=models.CASCADE)
     sent = models.BooleanField(default=False)
+    token = models.CharField(max_length=64, default=generate_new_token, unique=True)
+
+    def aggregate_submissions(self):
```

Comment:
We should (not that important yet) write a few simple tests on this function. Would be a lot easier to refactor without manual testing then :slightly_smiling_face: